pkg/addons/client.go

```diff
@@ -1,23 +1,17 @@
 package addons
 
 import (
+	"context"
 	"errors"
-	"io"
 	"log"
-	"net"
-	"net/http"
-	"net/url"
-	"os"
 	"reflect"
-	"syscall"
-	"time"
 
 	"github.com/go-test/deep"
+	"github.com/openshift/openshift-azure/pkg/checks"
```

Comment:
we seem to have the style of keeping the github.com/openshift/openshift-azure imports in a separate block - separate for consistency